src/Promitor.Agents.Core/Validation/Steps/AzureAuthenticationValidationStep.cs

```diff
@@ -21,16 +22,41 @@ public AzureAuthenticationValidationStep(IConfiguration configuration, ILogger<A
 
         public ValidationResult Run()
         {
-            var applicationId = _configuration.GetValue<string>(EnvironmentVariables.Authentication.ApplicationId);
-            if (string.IsNullOrWhiteSpace(applicationId))
+            var authenticationConfiguration = _configuration.GetSection("authentication").Get<AuthenticationConfiguration>();
```

Comment:
Some nits for this method:
- Can you add `{}` to the ifs, even if it's a oneliner?
- You don't need to prefix it with `this.`

Reply:
Ah ah, I'm used to use `this` everytime :) 
I did too much TypeScript I guess !

Reply:
Haha that's ok, I'm just being a pain :D